src/System.Private.Xml/tests/XmlResolver/System.Xml.XmlResolver.Tests/XmlPreloadedResolverGetEntity.cs

```diff
@@ -159,11 +159,11 @@ private static IEnumerable<(string PublicId, string SystemId, string SourcePath)
                 ("-//Netscape Communications//DTD RSS 0.91//EN", "rss-0.91.dtd"),
             };
 
-            string dtdFolderRoot = @"Utils\DTDs\";
+            string dtdFolderRoot = "Utils/DTDs/";
 
             return Enumerable.Concat(
-                GetKnownDtds(xhtmlDtds, "http://www.w3.org/TR/xhtml1/DTD/", Path.Combine(dtdFolderRoot, @"XHTML10\no_comments\")),
-                GetKnownDtds(rssDtds, "http://my.netscape.com/publish/formats/", Path.Combine(dtdFolderRoot, @"RSS091\no_comments\")));
+                GetKnownDtds(xhtmlDtds, "http://www.w3.org/TR/xhtml1/DTD/", Path.Combine(dtdFolderRoot, "XHTML10/no_comments/")),
+                GetKnownDtds(rssDtds, "http://my.netscape.com/publish/formats/", Path.Combine(dtdFolderRoot, "RSS091/no_comments/")));
```

Comment:
I meant i.e.: `Path.Combine(dtdFolderRoot, "RSS091", "no_comments")`

Reply:
Sorry, I didn't actually see your comment until after I'd attempted to fix the build failure. I'll push another commit.